### DISCUSSION TITLE
how do I prevent it from doing this

### DISCUSSION THREAD

**Discussion by daveandbambifan1234** [JordanSantiagoYT/FNF-JS-Engine#786] [General]
https://gamebanana.com/tools/9651
so I used the vertical bar script and it caused this problem

https://github.com/user-attachments/assets/488ab7df-bd1c-46a2-8066-e91984eb4f35

**Comment by daveandbambifan1234** [comment]
oops let me retake the video and remove my character

**Comment by daveandbambifan1234** [comment]
retake:
https://github.com/user-attachments/assets/2448ce71-8a62-4f34-937a-eecabd7f33a9

**Comment by TheStinkern** [comment]
look into the script that has the black borders and look for something about a font

remove all font related things

**Reply by daveandbambifan1234** [reply]
ok

**Reply by daveandbambifan1234** [reply]
show me where to delete that cuz I'm bad at coding

**Reply by TheStinkern** [reply]
well..
it's "setTextFont" I think

**Reply by TheStinkern** [reply]
I didn't even see the health bar move

**Reply by daveandbambifan1234** [reply]
> I didn't even see the health bar move
https://github.com/user-attachments/assets/2448ce71-8a62-4f34-937a-eecabd7f33a9
you should have clicked this video link, then you'll know exactly what happend

**Reply by daveandbambifan1234** [reply]
I retook the video for a reason

**Comment by JordanSantiagoYT** [comment]
Maybe.. idk UPDATE YOUR JS ENGINE??

i get that this is rude but you've been making tons of issues THAT HAVE ALREADY BEEN FIXED IN FUTURE UPDATES

**Reply by daveandbambifan1234** [reply]
1. I do not use the newer versions because it's laggier for my computer 
2. that's not only that I don't use psych engine 1.0 note features if you get what I'm saying

**Reply by TheStinkern** [reply]
there's (I guess) no other way then

**Reply by moxie-coder** [reply]
> 1. I do not use the newer versions because it's laggier for my computer
> 2. that's not only that I don't use psych engine 1.0 note features if you get what I'm saying

then get a better computer for christmas, idk what to tell you but you've been making many *many* issues related to already fixed issues in **newer versions**
